### ISSUE TITLE
Make a value of 'imports' property of py_library rule visible in Skylark

### ISSUE THREAD

**Issue by nuh-temp** [bazelbuild/bazel#2617]
In most cases that parameter indicates a path to library's sources and widely used along with `new_[git|local]_repository` rules in order to setup import path for external python libraries.
In some cases all such libraries should go through additional processing, like for example, if library is used with AppEngine you need to put it into a folder which is known to [AppEngine](https://cloud.google.com/appengine/docs/standard/python/tools/using-libraries-python-27#installing_a_third-party_library) to be able to import it properly.

**Comment by hermione521** [comment]
@lberki what do you think?

**Comment by evanj** [comment]
I just ran into this while attempting to figure out how to build a Python mono-repo that puts separate Python packages in different sub-directories. We would like to be able to use PEX, since that can package pre-built wheels which Bazel doesn't support yet. While trying to figure out how to write custom rules to support this, I discovered that I can't because I can't access `imports` from Skylark. See https://github.com/benley/bazel_rules_pex/issues/36 for more motivation.
